Source/WebKitLegacy/mac/DOM/DOMImplementation.mm

```diff
@@ -59,7 +59,7 @@ - (void)dealloc
 
 - (BOOL)hasFeature:(NSString *)feature version:(NSString *)version
 {
-    return WebCore::SVGTests::hasFeatureForLegacyBindings(feature, version);
+    return true;
```

Comment:
We're in Objective-C, so this should `return YES`

Reply:
Done in https://github.com/WebKit/WebKit/commit/8e33b21f45a92e8cc4b1e9d2efee26998e364e14.